benchopt/plotting/generate_html.py

```diff
@@ -230,24 +230,27 @@ def shape_solvers_for_html(df, objective_column):
 
         color, marker = get_solver_style(solver)
         # to preserve support of previous benchopt version
-        # where 'stopping_strategy' wasn't saved in solver meta
+        # where 'sampling_strategy' wasn't saved in solver meta
         try:
```

Comment:
I am not super fan of using nested try except as this makes the code hard to read.

I would be in favor of using a `if` checking if the column name exists.